doc/nrf/libraries/networking/aws_iot.rst

```diff
@@ -13,39 +13,179 @@ The library supports the following technologies:
+
+      #. Obtain a signed certificate using
+
+         .. code-block:: console
+
+            aws iot create-certificate-from-csr --certificate_signing_request device_cert.csr.pem --certificate-pem-outfile device_cert.pem --set-as-active
+
+      #. Take note of the certificate ARN from the previous command, as it is required later.
+      #. Provision the certificate using
+
+         .. code-block:: console
+
+            nrfcredstore <serial port> write <sec tag> CLIENT_CERT device_cert.pem
+
+         where ``<serial port>`` is the serial port of your device and ``<sec tag>`` is the same security tag used to obtain the CSR.
+
+      #. Download the `Amazon Root CA 1`_ PEM file.
+      #. Provision the certificate using
+
+         .. code-block:: console
+
+            nrfcredstore <serial port> write <sec tag> ROOT_CA_CERT AmazonRootCA1.pem
+
+         where ``<serial port>`` is the serial port of your device and ``<sec tag>`` is the same security tag used to obtain the CSR.
+
+   .. tab:: nRF91: Keys generated by AWS
+
+      .. warning::
+         This option is not recommended for production scenarios, since the private key leaves the device.
+
+      .. important::
+         The :ref:`at_client_sample` sample has to be flashed onto the device before following this guide.
+
+      To obtain a key pair and certificate generated by AWS and provision them to the modem, perform the following steps:
 
-.. _creating_a_thing_in_AWS_IoT:
+      1. Generate the key pair and certificate with
 
-Create a Thing in AWS IoT
-*************************
+         .. code-block:: console
 
-Before you can use this library, you must create a *Thing* for your client in AWS IoT.
-The *Thing* must be connected to a security policy.
-For testing, you can use a permissive policy, but make sure to update the policy to be more restrictive before you go into production.
-See `AWS IoT Developer Guide: Basic Policy Variables`_ and `AWS IoT Developer Guide: Security Best Practices`_ for more information about policies.
+            aws iot create-keys-and-certificate --set-as-active --certificate-pem-outfile device_cert.pem --public-key-outfile pub_key.pem --private-key-outfile priv_key.pem
 
-To create a Thing for your device:
+      #. Take note of the certificate ARN from the previous command, as it is required later.
+      #. Obtain a list of installed keys with
 
-1. Log in to the `AWS IoT console`_.
-#. Go to :guilabel:`Security` > :guilabel:`Policies` and select :guilabel:`Create policy`.
-#. Enter a name and define your policy.
-   For testing purposes, you can use the following policy (click on :guilabel:`JSON`, copy, and paste it):
+         .. code-block:: console
+
+            nrfcredstore <serial port> list
+
+         where ``<serial port>`` corresponds to the serial port of your device.
+
+      #. Choose a security tag that is not yet in use and thus does not show up in the list of installed keys.
```

Comment:
There is a section specifically about setting the Kconfig options further down in the docs, so I'm not sure whether we need to mention that here as well?

Reply:
I just thought it would be nice to explicitly state that the numbers need to match with the kconfig option. If the user haven't configured the Kconfig options yet they might not have thought about the sec tag yet and be confused what to put there. By explaining what kconfig option to use I think it should be easier for them to find the relevant information when they first need a sec tag.